modules/yieldmoBidAdapter.js

```diff
@@ -92,6 +94,20 @@ export const spec = {
         }
       });
       serverRequest.p = '[' + serverRequest.p.toString() + ']';
+
+      // check if url exceeded max length
+      const url = `${BANNER_SERVER_ENDPOINT}?${utils.parseQueryStringParameters(serverRequest)}`;
+      let extraCharacters = url.length - MAX_BANNER_REQUEST_URL_LENGTH;
+      if (extraCharacters > 0) {
+        for (const property of BANNER_REQUEST_PROPERTIES_TO_REDUCE) {
```

Comment:
Not sure our transpiler handles the `for... of ` for IE. So some tests will fail in IE I am guessing.

So please update this to use ES5 if possible!

Reply:
I think our transpiler will handle `for... of` loop even for IE. Looking at the build I see that this loop was changed to:

```
var _iterator = _createForOfIteratorHelper(BANNER_REQUEST_PROPERTIES_TO_REDUCE),
    _step;

try {
  for (_iterator.s(); !(_step = _iterator.n()).done;) {
    var property = _step.value;
    extraCharacters = shortcutProperty(extraCharacters, serverRequest, property);

    if (extraCharacters <= 0) {
      break;
    }
  }
} catch (err) {
  _iterator.e(err);
} finally {
  _iterator.f();
}
```

But, I would also suggest, if possible, not to use `for... of` loop, simply because built `.js` file would have less code and it would be a bit smaller.